frontend/ui/widget/filechooser.lua

```diff
@@ -235,16 +235,18 @@ local FileChooser = Menu:extend{
                     local doc_settings = DocSettings:open(item.path)
                     local summary = doc_settings:readSetting("summary")
 
-                    -- books marked as "finished" should be considered the same as 100%
+                    -- books marked as "finished" or "on hold" should be considered the same as 100% and 0% respectively
                     if summary and summary.status == "complete" then
-                        item.percent_finished = 1.0
-                        return
+                        item.sort_percent = 1.0
```

Comment:
What is the purpose of the decimal point?
`sort_percent` here can be local, declared together with `percent_finished`.

Reply:
>  sort_percent here can be local, declared together with percent_finished.

addressed with 14d027a0362c4cdc6aef3027f18104743309ae2e

> What is the purpose of the decimal point?

it was there before this PR, so it was only a property rename. the original purpose was because i didnt know if lua needed a explicit `.0` for float numbers (like some other languages), but also just to indicate a flat `1.0`